api/proto/config/storage.proto

```diff
@@ -123,7 +126,9 @@ message ContentTree {
   // The sha is for the top blob of the ContentTree
   string sha256 = 5;
 
-  uint64 maxSizeBytes = 6; // used for capping resource consumption in EVE
+  // maxSizeBytes used for capping resource consumption in EVE.
+  // maxSizeBytes indicates the maximum download size of content tree.
```

Comment:
Can this be zero=unlimited?

Reply:
Yes, size can be zero for the container's content tree but not for the VMs.

Reply:
Or we can enhance our logic in EVE that if the size is zero in the config then it will be considered as an unlimited download size for both the containers and VMs.

Reply:
I think zero should mean the same for VMs and containers. We can fix the code later but it makes sense that these comments anticipate those code fixes.

The other thing which probably makes sense to add for the comments is a 'unlimited' will still be limited by the device based on its available space.